docs/html/reference/requirements-file-format.md

```diff
@@ -171,15 +170,10 @@ This is will not work with other build-backends or newer setup.cfg-only projects
 
 If you have a declaration like:
 
-    FooProject >= 1.2 --global-option="--no-user-cfg" \
-                      --install-option="--prefix='/usr/local'" \
-                      --install-option="--no-compile"
+    FooProject >= 1.2 --global-option="--no-user-cfg"
 
 The above translates roughly into running FooProject's `setup.py` script as:
 
-    python setup.py --no-user-cfg install --prefix='/usr/local' --no-compile
+    python setup.py --no-user-cfg install
```

Comment:
Perhaps this `install` should be changed to `bdist_wheel`? I'm assuming that `pip` won't be using `install` anymore if `--install-option` is not used.

Reply:
@SpecLad yes, that is correct. I'll take care of this. Thanks!

Reply:
Hm, actually that will be for a subsequent PR that removes the setup.py install code path. This one just removes `--install-option`.

Reply:
I know that the `setup.py install` code path still exists, but I think in this example it's reasonable to assume that it won't be used in the absence of `--install-option`.